modules/core/src/test/java/org/apache/ignite/internal/GridTaskFutureImplStopGridSelfTest.java

```diff
@@ -83,7 +81,7 @@ public void testGet() throws Exception {
         try {
             final ComputeTaskFuture<?> fut = executeAsync(ignite.compute(), GridStopTestTask.class.getName(), null);
 
-            fut.listen((CI1<IgniteFuture>)gridFut -> {
+            fut.listen(ignored -> {
```

Comment:
ignored -> () ?

Reply:
It's an IgniteFuture (which is a Public API) that was not changed at this PR.

Reply:
Rolled back :(